patches/@snapshot-labs+lock+0.1.100.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/node_modules/@snapshot-labs/lock/connectors/walletconnect.ts b/node_modules/@snapshot-labs/lock/connectors/walletconnect.ts
```

Comment:
What do we need this patch for?

Reply:
This is my attempt to fix the issue `require is not defined` but it's not working

Reply:
Ahh I see